.drone.yml

```diff
@@ -29,3 +30,59 @@ steps:
 
 image_pull_secrets:
   - dockerconfigjson
+
+---
+kind: pipeline
+name: packer
+
+steps:
+- name: Build DigitalOcean Opendax image
+  image: appleboy/drone-packer
+  environment:
+    DIGITALOCEAN_API_TOKEN:
+      from_secret: digitalocean_api_token
+  commands:
+    - packer build ./packer/do-packer.json
+
+- name: Build AWS Opendax image
+  image: appleboy/drone-packer
+  environment:
+    AWS_ACCESS_KEY:
+      from_secret: aws_access_key
+    AWS_SECRET_KEY:
+      from_secret: aws_secret_key
+  commands:
+    - packer build ./packer/aws-packer.json
+
+- name: Create google credential
+  image: alpine:3.8
+  environment:
+    JSON_CREDENTIAL:
+      from_secret: json_credential
+  commands:
+    - echo "$JSON_CREDENTIAL" > cred.json
+
+- name: Build GKE Opendax base image
+  image: hashicorp/packer
+  failure: ignore
+  environment:
+    GOOGLE_APPLICATION_CREDENTIALS: cred.json
+    PROJECT_ID:
+      from_secret: project_id
+  commands:
+    - packer build -var project_id=$PROJECT_ID ./packer/gcp-image-base.json
+
+- name: Build GKE Opendax image
+  image: hashicorp/packer
+  environment:
+    GOOGLE_APPLICATION_CREDENTIALS: cred.json
+    PROJECT_ID:
+      from_secret: project_id
+  commands:
+    - packer build -var project_id=$PROJECT_ID ./packer/gcp-packer.json
```

Comment:
Let's try to build all of them in parallel pipelines like here https://github.com/openware/peatio/blob/master/.drone.yml